### ISSUE TITLE
how to lora finetune the causal casvid ?

### ISSUE THREAD

**Issue by leinyun-1** [tianweiy/CausVid#36]
I imitate ode initialization in origin paper to fine-tune the network by lora.  i use the denoising list of [1000,757,522,0] to add noise to clean latent from my dataset by the diffusion forcing strategy, and take clean latent as the target.  All configurations have not been changed.  But my loss curve and test result are bad .  Are there anybody successfully make lora finetune?  H

https://github.com/user-attachments/assets/7d705f5f-c625-490f-b5f3-b4b88c3c3d8a

elp me please.

**Comment by leinyun-1** [comment]
i'm based on the ckpt/autoregressive_checkpoint/model.pt , which is casual causvid
